docs/concepts/artifacts.md

```diff
@@ -225,6 +225,17 @@ prefect artifact ls
 
 You can also use the [Prefect REST API](https://app.prefect.cloud/api/docs#tag/Artifacts/operation/read_artifacts_api_accounts__account_id__workspaces__workspace_id__artifacts_filter_post) to programmatically filter your results.
 
+### Fetching artifacts
+
+In Python code you can retrieve an artifact stored on the server with the `Artifact.get` class method:
```

Comment:
nit: to avoid mildly confusing cloud-only users
```suggestion
In Python code, you can retrieve an existing artifact with the `Artifact.get` class method:
```